### PR TITLE
Fix disabled proxy setting in RSA NetWitness

### PR DESCRIPTION
**Pull request by omercnet**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15413

## Description
Fixing no proxy setting to disable proxy

## Screenshots
image
image

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Integrations/integration-RSANetWitness_v11.1.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ script:
                 'https': https
             }
             return proxies
-        return None
+        return {"http": None, "https": None}
```
Comment (Itay4) [inline Integrations/integration-RSANetWitness_v11.1.yml]: we should also delete proxy environment vars in this case

Comment (omercnet) [reply Integrations/integration-RSANetWitness_v11.1.yml]: if you really want you can use this http://docs.python-requests.org/en/master/api/#requests.Session.trust_env

but it's an overkill.. this explicitly tells requests not to use a proxy

Comment (Itay4) [reply Integrations/integration-RSANetWitness_v11.1.yml]: I meant:
```
    del os.environ['HTTP_PROXY']
    del os.environ['HTTPS_PROXY']
    del os.environ['http_proxy']
    del os.environ['https_proxy']
```

Comment (omercnet) [reply Integrations/integration-RSANetWitness_v11.1.yml]: but why?

**Comment by anara123** [comment]
Because this is the way we handle proxy in all the integrations

Sent from my iPhone

On 5 Feb 2019, at 18:42, Omer Cohen <notifications@github.com<mailto:notifications@github.com>> wrote:


@omercnet commented on this pull request.

________________________________

In Integrations/integration-RSANetWitness_v11.1.yml<https://github.com/demisto/content/pull/2947#discussion_r253947033>:

> @@ -83,7 +83,7 @@ script:
                 'https': https
             }
             return proxies
-        return None
+        return {"http": None, "https": None}


but why?

**Comment by omercnet** [comment]
> Because this is the way we handle proxy in all the integrations

so what you're saying is that you want to keep doing things wrong? gotcha :)

**Comment by anara123** [comment]
@omercnet what is the exact problem with
```
if not demisto.params()['proxy']:
        del os.environ['HTTP_PROXY']
        del os.environ['HTTPS_PROXY']
        del os.environ['http_proxy']
        del os.environ['https_proxy']
```

**Pull request merged**